android/BUILD.gn

```diff
@@ -14,6 +14,7 @@ declare_args() {
 
 java_cpp_enum("brave_android_java_enums_srcjar") {
   sources = [
+    "//brave/browser/android/constants.h",
```

Comment:
this file should have it's own `java_cpp_enum` in https://github.com/brave/brave-core/pull/24164/files#diff-c528ac841614c7466a1d915247b27506c4318d8bea37e40f0c63b186e24b28a0

The decentralized_dns one should also be fixed in a follow-up.